src/detect-transform-base64.c

```diff
@@ -149,15 +142,17 @@ static void TransformFromBase64Decode(InspectionBuffer *buffer, void *options)
         decode_length = nbytes;
     }
 
+    uint8_t decoded[input_len];
+    uint32_t num_decoded = 0;
+
     // PrintRawDataFp(stdout, input, input_len);
-    uint32_t decoded_length = 0;
-    uint32_t consumed = 0;
-    Base64Ecode code =
-            DecodeBase64(output, input_len, input, decode_length, &consumed, &decoded_length, mode);
-    if (code != BASE64_ECODE_ERR) {
-        // PrintRawDataFp(stdout, output, decoded_length);
-        if (decoded_length) {
-            InspectionBufferCopy(buffer, output, decoded_length);
+    bool b64decoded = Base64Decode(
```

Comment:
Should we use only one argument for the output size ?
Passed as a pointer with the buffer length, and modified by Base64Decode to be the number of bytes actually decoded

Reply:
ok